### PR TITLE
Added homework and README.md file

### PR DESCRIPTION
**Pull request by nare99**
1. Added pascal.py file, which structs n rows of pascal triangle
2. Added perfect.py file, which checks if given number is perfect or not
3. Added fibonachi.py file, which returns n'th fibonachi number
4. Added README.md file

### CODE REVIEW

File: Nare_Mkrtchyan/python/25_05/pascal.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            list2=[]
+        return
+
+print("Enter row number to get first n rown of Pascal triangle: ")
+num=input()
+if check(num):
+    num=int(num)
+    pascal(num,list2)
+else:
+    print(num,"is not number. Enter number.")
```
Comment (ani-im) [inline Nare_Mkrtchyan/python/25_05/pascal.py]: 1. image
Validation for values (negative, big number (non user-friendly))

2. Can be optimized

Comment (ani-im) [reply Nare_Mkrtchyan/python/25_05/pascal.py]: liked coding style and doc-strings :)

File: Nare_Mkrtchyan/python/25_05/perfect.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    else:
+        print(n,"is not perfect number")
+
+print("Enter number to check if it is perfect or not:")
+num=input()
+if check(num):
+    num=int(num)
+    perf(num)
+else:
+    print(num,"is not number. Enter number.")
```
Comment (ani-im) [inline Nare_Mkrtchyan/python/25_05/perfect.py]: 1. image

2. Perfect number is a positive integer
image

3. mysum -> coding style - under_scores for variables, add spaces please

4. limitation
image

File: Nare_Mkrtchyan/python/25_05/fibonachi.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    """
+    num=input()
+    if check(num):
+        num=int(num)
+        print(num,"'th fibonachi item is: ", fibonachi(num))
+    else:
+        print(num,"is not number. Enter number.")
+        start()
+
+start()
```
Comment (ani-im) [inline Nare_Mkrtchyan/python/25_05/fibonachi.py:60]: 1. image

2. F(0) -> 0
F(1) -> 1 etc
https://en.wikipedia.org/wiki/Fibonacci_number


image

3. 
image

**Review by ani-im — CHANGES_REQUESTED** [review]

File: Nare_Mkrtchyan/python/25_05/pascal.py (hunk 1 above)
Comment (nare99) [reply Nare_Mkrtchyan/python/25_05/pascal.py]: Validation done

File: Nare_Mkrtchyan/python/25_05/perfect.py (hunk 2 above)
Comment (nare99) [reply Nare_Mkrtchyan/python/25_05/perfect.py]: Tried to do marked changes.

File: Nare_Mkrtchyan/python/25_05/fibonachi.py (hunk 3 above)
Comment (nare99) [reply Nare_Mkrtchyan/python/25_05/fibonachi.py:60]: Tried to do marked changes.

**Pull request merged**